### PR TITLE
Update for Django 4

### PR DESCRIPTION
**Pull request by marcin-ro**
Update the library to work with Django 4.2.

### CODE REVIEW

File: aa_stripe/signals.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 import django.dispatch
 
-stripe_charge_succeeded = django.dispatch.Signal(providing_args=["instance"])
-stripe_charge_card_exception = django.dispatch.Signal(providing_args=["instance", "exception"])
-stripe_charge_refunded = django.dispatch.Signal(providing_args=["instance"])
+stripe_charge_succeeded = django.dispatch.Signal()
+stripe_charge_card_exception = django.dispatch.Signal()
+stripe_charge_refunded = django.dispatch.Signal()
```
Comment (remik) [inline aa_stripe/signals.py:6]: A bit of change. Is this working with 3.x too?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
```
Comment (remik) [inline .github/workflows/tests.yml:11]: Please update setup.py

File: aa_stripe/signals.py (hunk 1 above)
Comment (marcin-ro) [reply aa_stripe/signals.py:6]: The `providing_args` was a no-op in 2.2 ("This is purely documentational, however, as there is nothing that checks that the signal actually provides these arguments to its listeners."), and in 3.2: absent from the docs and deprecated in code.

Comment (marcin-ro) [reply aa_stripe/signals.py:6]: But thanks for looking closely! :)

File: .github/workflows/tests.yml (hunk 2 above)
Comment (marcin-ro) [reply .github/workflows/tests.yml:11]: Good catch, thanks!

**Review by remik — APPROVED** [review]

**Pull request merged**